cahee/index.html

```diff
@@ -10,13 +10,34 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="css/styles.css" />
+    <link rel="stylesheet" href="css/styles_tablet.css" />
+    <link rel="stylesheet" href="css/styles_mobile.css" />
+
     <script
       src="https://kit.fontawesome.com/7e9a77bb62.js"
       crossorigin="anonymous"
     ></script>
-    <title>Home work 12. Task 1</title>
+    <title>Home work 13. Task 1</title>
   </head>
   <body>
+    <svg display="none">
```

Comment:
Can we export/import as an image or SVG-sprite? It's pretty big in HTML :( Like you did for #globe

Reply:
Fixed.